cmd/kube-apiserver/apiserver.go

```diff
@@ -30,7 +31,12 @@ import (
 )
 
 func main() {
-	rand.Seed(time.Now().UnixNano())
+	var b [8]byte
+	_, err := cryptorand.Read(b[:])
```

Comment:
there are ~9 instances of this already. if this is worth doing, we should put this in a `func InitMathRand() error` function and call it consistently from all locations.

Reply:
With many occurrences to be replaced, do you have suggestion where the InitMathRand should reside ?

thanks